package.json

```diff
@@ -85,6 +85,6 @@
     "docs:watch": "gulp watch site --dev",
     "test:fast": "gulp karma-fast",
     "test:full": "gulp karma",
-    "lint": "eslint src"
+    "lint": "eslint src docs gulp config scripts"
```

Comment:
could probably be `eslint .`

Reply:
Would that try to lint `dist/`?

Reply:
I believe eslint also uses `.gitignore ` by default.

Reply:
ah, it looks like you can't set two ignore files. Probably worth just adding dist to .eslintignore.

Reply:
https://github.com/eslint/eslint/issues/9794

Reply:
Updated and pushed.